_posts/2021-06-03-opensearch-release-candidate-announcement.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title:  "The OpenSearch Release Candidate is now available on GitHub"
+authors: 
+  - andhopp
+date:   2021-06-02
+categories: update
+---
+
+Today we are proud to announce the Release Candidate for version 1.0.0 of both OpenSearch (derived from Elasticsearch 7.10.2) and OpenSearch Dashboards (derived from Kibana 7.10.2). The Release Candidate includes downloadable [artifacts](https://opensearch.org/downloads.html) (Linux tars and Docker images) as well as a number of OpenSearch plugins (listed below). 
+
+**So what does a release candidate mean exactly?**
+
+A Release Candidate is a beta version of a project that is feature complete and passing automated testing with the intent to validate expected functionality before moving to a GA launch. The goal with this Release Candidate is to share the current release with the community and solicit a final round of testing and feedback before the GA release. Between now and the 1.0.0 release of OpenSearch and OpenSearch Dashboards, we expect no further changes apart from critical bug fixes and the release of additional artifacts (see below). 
+
+**What is included in this Release Candidate?**
```

Comment:
I would add a paragraph that explains new feature highlights that are captured in the RC1 column of [the roadmap](https://github.com/orgs/opensearch-project/projects/1) in addition to all the release notes.

Reply:
I'll start digging into the release notes, thanks!